.github/workflows/dev_labeler.yml

```diff
@@ -0,0 +1,31 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+on:
+- pull_request_target
+
+jobs:
+  assign-rust-labels:
+    if: github.repository == 'apache/arrow'
```

Comment:
We don't need this restriction.
We have this restriction in `dev_cron.yml` because cron GitHub Actions logs in forks hide other logs.
`pull_request_target` doesn't run this job periodically. So we don't need this restriction.